php/octo-facts/.env.dev

```diff
@@ -0,0 +1,64 @@
+APP_NAME=Laravel
+APP_ENV=local
+APP_KEY=base64:6EtKvaRrx9y2zOPiDOxA+43ATjY5YmkYKWjKArLbU5c=
```

Comment:
Is this sensitive?

Reply:
not really. this key is auto generated when we create a new Laravel app, and it can be regenerated any time. the general rule of thumb is to use a separate .env file for production, that's why we don't have a `.env` directly on the repo, the user needs to copy this `.env.dev` file for setting up a dev environment.